docs/src/concepts.md

```diff
@@ -0,0 +1,116 @@
+| Clustering | specify instance(s) of clustering on an `EphysRecording`, by some `ClusteringMethod`. |
+| Curation | specify instance(s) of curations performed on the output of a given `Clustering`. |
+| CuratedClustering | set of results from a particular round of clustering/curation. |
+| CuratedClusting.Unit | Identified unit(s) from one `Curation`, and the associated properties (e.g. cluster quality, spike times, spike depths, etc.). |
+| WaveformSet | A set of spike waveforms for units from a given `CuratedClustering`. |
+
+## Element Development
+
+Through our interviews and direct collaboration on the precursor projects, we identified the common motifs to create the [Array Electrophysiology Element](https://github.com/datajoint/element-array-ephys).
+
+Major features of the Array Electrophysiology Element include:
+
++ Pipeline architecture detailed by:
+
+    + Probe, electrode configuration compatible with Neuropixels probes and generalizable to other types of probes (e.g. tetrodes) - supporting both `chronic` and `acute` probe insertion modes.
+
+    + Probe-insertion, ephys-recordings, LFP extraction, clusterings, curations, sorted units and the associated data (e.g. spikes, waveforms, etc.).
+    
+    + Store/track/manage different curations of the spike sorting results - supporting both curated clustering and kilosort triggered clustering (i.e., `no_curation`).
+
++ Ingestion support for data acquired with SpikeGLX and OpenEphys acquisition systems. 
++ Ingestion support for spike sorting outputs from Kilosort.
++ Triggering support for workflow integrated Kilosort processing.
++ Sample data and complete test suite for quality assurance.
+
+Incorporation of SpikeInterface into the Array Electrophysiology Element will be on DataJoint Elements development roadmap. Dr. Loren Frank has led a development effort of a DataJoint pipeline with SpikeInterface framework and NeurodataWithoutBorders format integrated (https://github.com/LorenFrankLab/nwb_datajoint).
+
+## Data Export and Publishing
+
+Element Array Electrophysiology supports exporting of all data into standard Neurodata Without Borders (NWB) files. This makes it easy to share files with collaborators and publish results on [DANDI Archive](https://dandiarchive.org/). [NWB](https://www.nwb.org/), as an organization, is dedicated to standardizing data formats and maximizing interoperability across tools for neurophysiology. For more information on uploading NWB files to DANDI within the DataJoint Elements ecosystem, visit our documentation for the DANDI upload feature of [Element Array Electrophysiology](datajoint.com/docs/elements/element-array-ephys/).
+
+## Roadmap
+
+Further development of this Element is community driven. Upon user requests we will continue adding features to this Element.
```

Comment:
@sidhulyalkar - Can you think of an example to add here? i.e. 'features to this Element, such as ...'

Reply:
Can add that we will plan to add functionality to support large (48 hour) neuropixel recordings via an overlapping segmented processing approach.